consensus/src/counters.rs

```diff
@@ -11,6 +11,10 @@ use aptos_metrics_core::{
 };
 use once_cell::sync::Lazy;
 
+pub const TXN_COMMIT_SUCCESS: &str = "success";
```

Comment:
nit: I'd probably add something like `LABEL` or `STATUS` to the name of these constants because when I read them from the other file, I thought they were counters 😄 e.g., `counters::TXN_COMMIT_SUCCESS` reads very much like a counter and not a string.